### DISCUSSION TITLE
https://angular.love/angular-scroll-position-restoration

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#364] [General]
# https://angular.love/angular-scroll-position-restoration

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/angular-scroll-position-restoration

**Comment by ebrahim-s-ebrahim** [comment]
I really appreciate you writing about this topic. Your insights on restoring scroll position is very helpful.

**Comment by intelligentRaji** [comment]
Thank you!

**Comment by rr-hornback** [comment]
Does this work for Angular Tables on a page as well and not just the overall page scroll position?

**Comment by greatkeke** [comment]
Thanks for your fantastic solution. That's great.

**Comment by jakehockey10** [comment]
I'm having a bit of trouble when the whole page isn't the scroll element.  I have a main element inside of my navigation component that surrounds the router-outlet.  How can I restore the scroll position of the main element on page navigation?
